src/Groups/homomorphisms.jl

```diff
@@ -390,11 +390,15 @@ Return an isomorphism from `G` to a group of type `T`.
 An exception is thrown if no such isomorphism exists.
```

Comment:
Perhaps we can also add an example/doctest to the docstring of this function. I wrote the patch below last night while working on PR #1246 but removed it again because I wanted that PR to focus. But perhaps we can add it (or a variant) now...

```suggestion
An exception is thrown if no such isomorphism exists.

If only the image of such an isomorphism is needed, use `T(G)`.

# Examples
```jldoctest
julia> G = dihedral_group(6)
<pc group of size 6 with 2 generators>

julia> iso = isomorphism(PermGroup, G)
Group homomorphism from
<pc group of size 6 with 2 generators>
to
Group([ (1,2)(3,6)(4,5), (1,3,5)(2,4,6) ])

julia> PermGroup(G)
Group([ (1,2)(3,6)(4,5), (1,3,5)(2,4,6) ])

julia> codomain(iso) == ans
true
```
````

Of course the final `codomain(iso) == ans` could be changed to `codomain(iso) === ans` with the changes in this PR.

Reply:
Good idea. Thanks.